erc20/README.md

```diff
@@ -39,20 +19,44 @@ The problem with a contract proxy is that the proxy does not actually call the i
 it loads the function code and executes it in it's own storage space. This means that the storage needs to be
 defined in the proxy as well, and the storage layout needs to be the same.
 
-![](erc20_setup.svg)
+![contract hierarchy diagram](erc20_setup.svg)
```

Comment:
Created with inkscape, there is an `inkscape svg` which supposedly has some extra data, but not sure if that is usefull for this simplistic design. Especially since the goal is to improve this and change this to a nice UML iirc, @robvanmieghem

Reply:
Why in inkscape, It's just a graph. I'll do it quickly in DOT and upload the source and new image later today. Already created an issue for it https://github.com/threefoldfoundation/tfchain/issues/259, and assigned it to me.